examples/Atlas/sdf/random_terrain.world

```diff
@@ -0,0 +1,498 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I agree, and I made the change

On Mon, Feb 2, 2015 at 5:46 PM, Robin Deits <notifications@github.com>
wrote:

> In examples/Atlas/sdf/random_terrain.world
> <https://github.com/RobotLocomotion/drake/pull/734#discussion_r23966544>:
>
> > @@ -0,0 +1,498 @@
> > +<?xml version="1.0" encoding="utf-8"?>
>
> Do we actually want to include this in the repository? If this is just a
> sample output from generateRandomBoxWorld.m then I think we probably
> don't need to include it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/734/files#r23966544>.
>